android/src/main/kotlin/com/airship/flutter/AirshipPlugin.kt

```diff
@@ -198,6 +206,64 @@ class AirshipPlugin : MethodCallHandler {
         result.success(UAirship.shared().pushManager.userNotificationsEnabled)
     }
 
+    private fun getActiveNotifications(result: Result) =
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                val notifications = arrayListOf<Map<String, Any>>()
+
+                val notificationManager = UAirship.getApplicationContext().getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+                val statusBarNotifications = notificationManager.activeNotifications
+
+                for (statusBarNotification in statusBarNotifications) {
+
+                    val notification = HashMap<String, Any>()
+                    notification.put("notification_id", statusBarNotification.id.toString())
```

Comment:
Looks like you are missing putting the original push body into the notification. Also seems broke on React.

This is how we accomplish it on Cordova - https://github.com/urbanairship/urbanairship-cordova/blob/master/src/android/CordovaNotificationProvider.java#L40
https://github.com/urbanairship/urbanairship-cordova/blob/master/src/android/UAirshipPlugin.java#L1045
https://github.com/urbanairship/urbanairship-cordova/blob/master/src/android/Utils.java#L31

Reply:
Right, so if follow the same logic and set the flutter notification provider like this:
https://github.com/urbanairship/urbanairship-cordova/blob/639efa3ee0945cbdce80b06aa52a64af9e8051e4/src/android/CordovaAutopilot.java#L66
then since it takes the config as parameter, does that mean that something similar to this should be added to flutter plugin?
https://github.com/urbanairship/urbanairship-cordova/blob/639efa3ee0945cbdce80b06aa52a64af9e8051e4/src/android/PluginManager.java#L307

Reply:
Nah, I think you can get away with just putting the payload in the extras for now. In FlutterAutopilot:
```
        airship.pushManager.notificationProvider = object : AirshipNotificationProvider(UAirship.getApplicationContext(), airship.airshipConfigOptions) {
            override fun onExtendBuilder(context: Context, builder: NotificationCompat.Builder, arguments: NotificationArguments): NotificationCompat.Builder {
                builder.extras.putBundle(PUSH_MESSAGE_BUNDLE_EXTRA, arguments.message.pushBundle)
                return super.onExtendBuilder(context, builder, arguments)
            }
        }
```